api/registrations/views.py

```diff
@@ -262,8 +262,9 @@ def get_default_queryset(self):
 
     def get_queryset(self):
         registration = self.get_node()
+        auth = get_user_auth(self.request)
         registration_pks = registration.node_relations.filter(is_node_link=False).select_related('child').values_list('child__pk', flat=True)
-        return self.get_queryset_from_request().filter(pk__in=registration_pks).can_view(self.request.user).order_by('-modified')
+        return self.get_queryset_from_request().filter(pk__in=registration_pks).can_view(auth.user).order_by('-modified')
```

Comment:
Was the error happening because `self.request.user` returned an `AnonymousUser`? If so, then perhaps `can_view` should be modified to handle that.